server/samples/clients/node/appencryption_client.js

```diff
@@ -0,0 +1,274 @@
+/*
+Asherah Server gRPC - NodeJS client
+
+A simple NodeJS application that demonstrates integrating with Asherah Server
+via a dynamically generated gRPC client.
+*/
+
+'use strict';
+
+const yargs = require('yargs');
+const grpc = require('@grpc/grpc-js');
+const protoLoader = require('@grpc/proto-loader');
+const sleep = require('sleep');
+
+// Setup logger
+const winston = require('winston');
+const consoleTransport = new winston.transports.Console();
+const myWinstonOptions = {
+    transports: [consoleTransport],
+    format:
+        winston.format.combine(
+            winston.format.colorize({all: true,}),
+            winston.format.simple()
+        ),
+};
+const logger = new winston.createLogger(myWinstonOptions);
+
+let requests = [];
+let protoPath = "";
+
+/**
+ * SessionClient provides a synchronous client interface for the bidirectionally-streaming Session
+ endpoint.
+ */
+class SessionClient {
+    #call;
+    #getSessionResolve;
+    #encryptResolve;
+    #decryptResolve;
+
+    constructor(callback) {
+        let appEncryptionDef = protoLoader.loadSync(__dirname + protoPath, {
+            keepCase: true,
+            defaults: true,
+            oneofs: true,
+        });
+
+        let appEncryptionProto = grpc.loadPackageDefinition(appEncryptionDef);
```

Comment:
Does every instance need to load the proto def? If not, consider passing as an argument instead.